src/development/LocalTest/Controllers/Storage/TextsController.cs

```diff
@@ -81,7 +81,6 @@ public async Task<ActionResult<TextResource>> Create(string org, string app, [Fr
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
         [ProducesResponseType(StatusCodes.Status404NotFound)]
         [Produces("application/json")]
-        [Authorize]
```

Comment:
I assume this must be added back in?

Reply:
No. We dont do any authorization in localtest

Reply:
We could make the argument to remove authorization on texts in Storage as well?

I don't like the idea of having different rules for LocalTest.